pom.xml

```diff
@@ -33,6 +34,7 @@
     <version.assertj>3.20.2</version.assertj>
     <version.jackson>2.13.1</version.jackson>
     <version.json>20230227</version.json>
+    <version.org.antlr4>4.10.1</version.org.antlr4>
```

Comment:
Do you think this version and Drools repo antlr versions can conflict or this is not possible?
In general if they _must_ stay aligned it might be useful to create a followup ticket to introduce the usage of `drools-parent` or similar to centralise the declaration.
Wdyt?

Reply:
Yes I think something like that in another PR 👍